### PR TITLE
chore(dialogs): add dialog docs

### PR DESCRIPTION
**Pull request by jknilans**
This PR adds docs related to modals and dialogs.

This should help close https://github.com/patternfly/patternfly-next/issues/1179

Updates include:

- screenshots now reflect left-aligned buttons
- added message dialog content to modal dialog documentation
- updated path

### CODE REVIEW

**Comment by jknilans** [comment]
@jschuler the build is breaking; not sure why. I thought the path change may have caused it so I reverted that change but it's still breaking. Thoughts?

File: packages/patternfly-4/content/design-guidelines/usage-and-behavior/modal/design.md [hunk 1]
```diff
@@ -21,24 +21,80 @@ The PatternFly components listed in the following sections can be used in a numb
...
+Use passive dialogs to communicate critical information like the status of an application or result of an action. The content of a passive dialog should be important, immediately relevant, and either lead to or require action on the user’s part.
+
+Some usage examples include:
+* Notifying a user that an operation may take a long time to complete
+* Notifying a user that a critical process is complete
+
+**Note:** If information is not critical, don’t use a passive dialog. Instead, use inline notifications or toast notifications.
+
+#### Get required user input
+Sometimes users may need to provide additional input in order to complete an action. In this situation a form or other type of element may be incorporated into the modal. An example of this use case is the [Modal Wizard](/documentation/core/demos/wizard).
```
Comment (mcarrano) [inline packages/patternfly-4/content/design-guidelines/usage-and-behavior/modal/design.md]: Not sure if I think this reference is needed here without more examples. @stacymcauliffe what do you think. I don't feel strongly about it, but seems that this specific document is now focused on modal messages and alerts.

**Comment by jschuler** [comment]
@jknilans you can ignore the netlify failures, as long as the travis succeeds it's good, thanks!

**Comment by mcarrano** [comment]
Will close this and recreate to merge into master (not gatsby2).

**Pull request closed without merging**